src/Middleware.php

```diff
@@ -60,9 +53,9 @@ public function version(Request $request)
     public function share(Request $request)
     {
         return [
-            'errors' => function () use ($request) {
+            'errors' => new AlwaysProp(function () use ($request) {
```

Comment:
Wondering if we should use `Inertia::always()` here instead of `AlwaysProp`? Just because I could see people viewing the source of this file often and I think we want to encourage `Inertia::always()` over `AlwaysProp`.

Reply:
Combined with the other change I'm suggesting, this could actually just be:

```php
'errors' => Inertia::always($this->resolveValidationErrors($request)),
```